contracts/identity/Identity.sol

```diff
@@ -104,6 +104,26 @@ contract Identity is ProxyStorage {
         emit TransactionExecution(hash, status);
     }
 
+    function executeOwnerTransaction(
+        address payable to,
+        uint256 value,
+        bytes memory data,
+        uint8 operationType
+    )
+        public
+    {
+        require(msg.sender == owner, "Only owner can call this");
+        bool status = applyOperation(
+            to,
+            value,
+            data,
+            operationType,
+            gasleft()
```

Comment:
That seems to me like it should not work, because it will then not have enough gas later for the check against gasLeft.

Reply:
Use uint(-1)